core/client/ui/toolboxes/level-toolbox.js

```diff
@@ -39,6 +45,30 @@ Template.levelToolbox.events({
     const { x, y } = user.profile;
     updateLevel(level.name, { x, y }, level.hide);
   },
+  'change .js-voice-amplifier-select'(event) {
```

Comment:
A small idea to have less code: add a `data-permission="xxx"` then on the JS side do something like:
```
'change .permission-list select'(event) {
    const { permission } = event.target.dataset;
    updateFeaturePermissionLevel(permission, event);
  },
```

Reply:
So cool, I searched for something like this but couldn't find, good to know :)